### PR TITLE
update #709 - header links are now only curriculum, repo, learn and community

### PR DESCRIPTION
**Pull request by pmulard**
- changed Journey to Learn
- changed Mattermost link to Discord
- removed Contributors

### CODE REVIEW

File: components/AppNav.tsx [hunk 1]
```diff
@@ -33,11 +33,14 @@ const navItems: NavItem[] = [
...
     path:
       'https://www.notion.so/Table-of-Contents-a83980f81560429faca3821a9af8a5e2',
-    name: 'Journey',
+    name: 'Learn',
     external: true
   },
-  { path: 'https://chat.c0d3.com', name: 'Help', external: true },
-  { path: '/contributors', name: 'Contributors' }
+  {
+    path: 'https://discord.com/channels/828783458469675019/',
```
Comment (ggwadera) [inline components/AppNav.tsx:40]: We have a custom discord URL for our server, you can use it here too :)

```suggestion
    path: 'https://discord.gg/c0d3',
```

**Review by ggwadera — COMMENTED** [review]
Hey @pmulard, thanks for another PR here for c0d3 :)

There's an unrelated test that is failing that I gotta check what's happening with it. Other than that, nice work 👍

File: components/AppNav.tsx (hunk 1 above)
Comment (pmulard) [reply components/AppNav.tsx:40]: Great, thank you

**Comment by ggwadera** [comment]
Ah your branch is pretty behind the master here, that's why the tests are failing. Run the commands below to get it up to date.

```bash
# add the upstream git remote if you haven't
git remote add https://github.com/garageScript/c0d3-app.git
# pull the commits from the master branch on upstream
git pull upstream master
```

**Comment by pmulard** [comment]
> Ah your branch is pretty far behind the master here, that's why the tests are failing. Run the commands below to get it up to date.
> 
> ```shell
> # add the upstream git remote if you haven't
> git remote add upstream https://github.com/garageScript/c0d3-app.git
> # pull the commits from the master branch on upstream
> git pull upstream master
> ```
> 
> If you have any conflicts you'll need to resolve them before commiting the merge commit.
> 
> After that update the snapshots again with `yarn test -u` as some of them are still outdated.

Ah I ran a git pull, but it must have only pulled from my forked repo and not garagescript's

**Review by ggwadera — APPROVED** [review]

**Review by songz — APPROVED** [review]

**Review by Ulisseus — APPROVED** [review]

**Comment by pmulard** [comment]
@ggwadera thank you for helping me troubleshoot this

**Pull request merged**